Flabbergasted-Flamingos/Bangazon-admin-app
# Delete future trainings

# Description
Add description about what this pull request covers.

## Number of Fixes
1. added TrainingsDeleteView, added optional button to trainings_list_view.html, added delete path to trainings. 

## Related Ticket(s)
closes #11 

## Problem to Solve
HR should be able to delete trainings that happen in the future.  


## Expected Behavior
while looking at the trainings list, a delete affordance should appear on training cards that end in the future, and manually deleting a card in the url should be forbidden. 

## Steps to Test Solution

```
git fetch --all
git checkout delete_future_trainings
```
remigrate
```
python manage.py seeder
```
go to training list, add a training that ends in the future.  Check to see that there is a delete button and that it works.  
in the url bar, try and delete a training (with its id) that already ended.  It should display 403 screen.
